addons/account/models/account_accrued_entry.py

```diff
@@ -0,0 +1,52 @@
+import json
+from odoo import models, _
+
+
+class AccruedEntryMixin(models.AbstractModel):
+    _name = 'account.accrued.entry.mixin'
+    _description = 'Account Accrued Entry Mixin'
+
+    def _get_preview_data(self, move_vals, currency_id=None):
+        preview_columns = [
+            {'field': 'account_id', 'label': _('Account')},
+            {'field': 'name', 'label': _('Label')},
+            {'field': 'debit', 'label': _('Debit'), 'class': 'text-end text-nowrap'},
+            {'field': 'credit', 'label': _('Credit'), 'class': 'text-end text-nowrap'},
+        ]
+        preview_vals = [self.env['account.move']._move_dict_to_preview_vals(move_vals, currency_id)]
+        return json.dumps({
+            'groups_vals': preview_vals,
+            'options': {
+                'columns': preview_columns,
+            },
+        })
+
+    def _get_aml_vals(self, label, balance, account_id, **kwargs):
+        values = {
+            'name': label,
+            'balance': balance,
+            'account_id': account_id,
+        }
+        values.update(kwargs)
+        return values
```

Comment:
```suggestion
    def _get_aml_vals(self, name, balance, account_id, **kwargs):
        values = {
            'name': name,
            'balance': balance,
            'account_id': account_id,
        }
        values.update(kwargs)
        return values
```
Consistency. Weird to call it `label` and then `name`.